Regression: webDAV-functionality broken on Windows

The following line was updated in version 2.7.0, but this breaks all webDAV-functionalities on Windows:

https://github.com/SalesforceCommerceCloud/sfcc-ci/blob/613bf7478afcba72daf3831ed1c0cbccc92a33c2/lib/webdav.js#L26

It uses path.join to join two path segments. However, path.join() is meant to interact with file paths, while this is an URI. This results in a path with backslashes on Windows, which isn't a valid URI. This can be seen by running the unit tests on Windows. All tests for lib/webdav.js fail due to these types of errors:

`AssertionError: expected '\\on\\demandware.servlet\\webdav\\Sites\\cartridges\\mycode.zip' to equal '/on/demandware.servlet/webdav/Sites/cartridges/mycode.zip'`

In the past, the path was just concatenated:
https://github.com/SalesforceCommerceCloud/sfcc-ci/blob/074cc74082c874bc5cc4f934ebd6d8964f186ad0/lib/webdav.js#L26

--- a/lib/webdav.js
+++ b/lib/webdav.js
@@ -23,7 +23,7 @@ if ( process.env.DEBUG ) {
 
 function getOptions(instance, path, token, options, method) {
     // the endpoint including the relative path on the instance's file system to upload to
-    var endpoint = require('path').join(WEBDAV_BASE, path)
+    var endpoint = WEBDAV_BASE + path;
 
     var opts = {
         baseUrl: 'https://' + instance,

